src/components/BeaconChainActions.tsx

```diff
@@ -69,36 +71,51 @@ export const DataLogo = graphql`
   }
 `
 
-const BeaconChainActions = () => {
-  const intl = useIntl()
-  const data = useStaticQuery(graphql`
-    query {
-      beaconscan: file(relativePath: { eq: "upgrades/etherscan.png" }) {
-        ...DataLogo
-      }
-      beaconchain: file(relativePath: { eq: "upgrades/beaconchainemoji.png" }) {
-        ...DataLogo
-      }
+type ChildImage = {
+  childImageSharp: {
+    gatsbyImageData: Record<string, unknown>
+  }
+}
+
+const BeaconStaticQuery = graphql`
+  query {
```

Comment:
In these cases, we need to give the query a name, like `query BeaconStatic {` in order to let gatsby generate the correct types. You build those query types by running `yarn develop`. You will find the new query type under this generated file: `src/gatsby-types.d.ts`.

Finally, you will be able to use them like this:
```ts
const data = useStaticQuery<Queries.BeaconStaticQuery>(BeaconStaticQuery)
```
Note that `Queries` is a global namespace.

Reply:
Ignore this since Gatsby is only generating the query types for the pages.